Platform/TigerlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1424,7 +1424,7 @@ UpdateFspConfig (
   }
   for (Index = 0; Index < 4; Index++) {
     FspsConfig->CpuPcieRpPtmEnabled[Index] = 0x1;
-    FspsConfig->PtmEnabled[Index] = 0x1;
+    FspsConfig->PtmEnabled[Index] = 0x0;
   }

```

Comment:
it looks line 1425 ~ 1428 could be removed since both will be set again in this function.

Reply:
noted and updated.